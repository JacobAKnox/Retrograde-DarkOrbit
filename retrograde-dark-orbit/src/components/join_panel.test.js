import { render } from '@testing-library/react'
import JoinPanel from './join_panel'
 
it('renders join panel unchanged', () => {
  const { container } = render(<JoinPanel/>)
  expect(container).toMatchSnapshot()
})