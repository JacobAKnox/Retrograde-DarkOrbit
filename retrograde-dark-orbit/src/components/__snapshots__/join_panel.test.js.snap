// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders join panel unchanged 1`] = `
<div>
  <div
    class="bg-inherit p-4 m-4 flex flex-col"
  >
    <input
      class="bg-slate-700 text-white m-1 py-2 px-10 rounded-xl"
      placeholder="Username"
    />
    <input
      class="bg-slate-700 text-white m-1 py-2 px-10 rounded-xl"
      placeholder="Code"
    />
    <button
      class="bg-slate-900 text-white m-1 py-2 px-10 rounded-xl"
    >
      Join
    </button>
  </div>
</div>
`;
